frontend_src/frontend/src/utils/utils_functions.py

```diff
@@ -93,41 +93,25 @@ def _load_protection_indicators_data(selected_country: str):
             st.session_state[f"protection_df_max_date_{selected_country}"] = "-"
 
 
-def _country_selection_filter(filter_name: str, country_index: Optional[int] = None):
+def _country_selection_filter(filter_name: str, disabled: Optional[bool]):
```

Comment:
Instead of Optional, use the default value: disabled: bool = False

Reply:
Okayy